Frontend/src/components/Input/DateInput.svelte

```diff
@@ -78,6 +79,7 @@
 </style>
 
 <Datepicker
+  bind:weekStart
```

Comment:
Since this does not change there is no need to bind here. You can just do weekStart={1}

Reply:
Ah, didn't know that! I tried `weekStart=1`, but that didn't do the trick. What is the difference to `weekStart={1}`? Why did `Datepicker.weekStart=1`? Would  `Datepicker.weekStart={1}` work?

Reply:
It has to be weekStart={1} because that's the correct svelte syntax (see https://svelte.dev/tutorial/declaring-props).

Reply:
ah, ok!